index.html

```diff
@@ -291,6 +291,16 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <div class="contact-container">
+      <h2 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h2>
+      <form action="https://formspree.io/f/xvodkydo" method="POST">
+        <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" required>
```

Comment:
- [ ] [required] Based on the mockup, it looks like the names fields should be divided into two separate fields: `first_name` and `last_name`, Would you fix this names field according to what the figma mockup has?

| Your version | Figma version |
| --- | --- |
| ![Screenshot from 2021-05-22 00-35-17](https://user-images.githubusercontent.com/56886064/119204394-9bb29e80-ba95-11eb-87b1-0514c3391a4a.png) | ![Screenshot from 2021-05-22 00-35-59](https://user-images.githubusercontent.com/56886064/119204436-b38a2280-ba95-11eb-8def-e90a97c21f7c.png)|

:point_right: Could you see the difference :question: :question: :question: